src/host/outputStream.cpp

```diff
@@ -279,11 +287,17 @@ void ConhostInternalGetSet::SetWorkingDirectory(const std::wstring_view /*uri*/)
 // - true if successful. false otherwise.
 void ConhostInternalGetSet::PlayMidiNote(const int noteNumber, const int velocity, const std::chrono::microseconds duration)
 {
+    const auto window = ServiceLocator::LocateConsoleWindow();
+    if (!window)

```

Comment:
Oh, I see. you prevent conhost from writing to the clipboard by THE FUNCTION NOT BEING IMPLEMENTED.

Reply:
FYI, we're tracking those TODO's in #13408.